ui/src/components/data-coverage/DataCoveragePage.module.scss

```diff
@@ -331,7 +334,407 @@
     border-radius: 50px;
     border: 2px solid var(--interactive-accent, #5FE500);
     margin-top: 30px;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 16px; /* 114.286% */
+    letter-spacing: 1.25px;
     cursor: pointer;
 }
 
 
+@media screen and (min-width: 360px) {
+    // style goes here
```

Comment:
If you aren't adding custom styles you can probably remove this breakpoint definiton